libkbfs/folder_block_manager.go

```diff
@@ -1161,18 +1169,29 @@ func (fbm *folderBlockManager) doReclamation(timer *time.Timer) (err error) {
 }
 
 func (fbm *folderBlockManager) reclaimQuotaInBackground() {
-	timer := time.NewTimer(fbm.config.QuotaReclamationPeriod())
+	timer := time.NewTimer(fbm.config.Mode().QuotaReclamationPeriod())
 	timerChan := timer.C
 	for {
 		// Don't let the timer fire if auto-reclamation is turned off.
-		if fbm.config.QuotaReclamationPeriod().Seconds() == 0 {
+		if fbm.config.Mode().QuotaReclamationPeriod().Seconds() == 0 {
 			timer.Stop()
 			// Use a channel that will never fire instead.
 			timerChan = make(chan time.Time)
```

Comment:
The doc suggests something like this:

```
if !t.Stop() {
	<-t.C
}
```

Not sure which way is better though.

Reply:
Thanks, but I don't think I want to mess with code that is working, in this PR anyway.